paymentsheet/src/androidTest/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <application>
         <activity
             android:name=".MainActivity"
+            android:theme="@style/Theme.AppCompat.Light"
```

Comment:
Why is this needed?

Reply:
Forgot to mention this. The `MainActivity` is used in `GooglePayButtonTest`, and the button’s progress indicator requires a `Theme.AppCompat` or descendent.